tree/ntuple/v7/inc/ROOT/RNTupleReadOptions.hxx

```diff
@@ -75,8 +75,8 @@ public:
    EImplicitMT GetUseImplicitMT() const { return fUseImplicitMT; }
    void SetUseImplicitMT(EImplicitMT val) { fUseImplicitMT = val; }
 
-   bool HasMetricsEnabled() const { return fEnableMetrics; }
-   void SetMetricsEnabled(bool enable) { fEnableMetrics = enable; }
+   bool HasMetricsEnabled() const { return fMetricsEnabled; }
+   void SetMetricsEnabled(bool val) { fMetricsEnabled = val; }
```

Comment:
Shouldn't it be `SetEnableMetrics()` to be in line with other verb + noun styles?

Reply:
I remember we discussed this but apparently I forgot what we settled on. I thought we want to carry the information that we don't enable metrics right away but we toggle the switch that eventually results in the metrics being enabled. But I may be wrong...

Reply:
Did we consider:
```
void EnableMetrics();
void DisableMetrics();
```
?
or
```
SetRecordMetrics(bool val);
```

Reply:
I unfortunately also don't remember exactly what we settled on, so I'll just write down some thoughts:
1. We didn't want to call it `EnableMetrics()` directly because it's called on the options and we're not actually operating on the options, but just record what we set after.
2. Instead we liked the `Set` prefix because it conveys that we remember to later call `EnableMetrics` on the reader once created.
3. Between `SetMetricsEnabled()` and `SetEnableMetrics()`, my argument for the latter is that it is already spelt this way for all write options: `SetUseBufferedWrite()`, `SetUseDirectIO()`, `SetUseImplicitMT()`, `SetEnablePageChecksums()`, `SetEnableSamePageMerging()`. We can change, but then we should probably be consistent everywhere else...

Reply:
Thanks! I think that makes sense!